src/renderers/WebGLRenderer.js

```diff
@@ -2371,10 +2372,26 @@ class WebGLRenderer {
 
 		};
 
-		this.copyTextureToTexture = function ( position, srcTexture, dstTexture, level = 0 ) {
+		this.copyTextureToTexture = function ( sourceBox, position, srcTexture, dstTexture, level = 0 ) {
 
-			const width = srcTexture.image.width;
-			const height = srcTexture.image.height;
+			if ( sourceBox.isBox2 !== true ) {
+
+				console.warn( 'WebGLRenderer: copyTextureToTexture function signature has changed.' );
+
+				position = arguments[ 0 ];
+				srcTexture = arguments[ 1 ];
+				dstTexture = arguments[ 2 ];
+				level = arguments[ 3 ];
+
+				const image = srcTexture.isCompressedTexture ? srcTexture.mipmaps[ level ] : srcTexture.image;
+				sourceBox = new Box2();
+				sourceBox.min.set( 0, 0 );
+				sourceBox.max.set( image.width, image.height );
+
+			}
```

Comment:
Presumably we'd want to remove this after 10 releases?

Reply:
Yes, that makes sense. It's best to add the annotation `// @deprecated, r165` so we can easier search for code that can be removed in future releases.